# SV

#### Nice papers

##### [High-resolution comparative analysis of great ape genomes](http://science.sciencemag.org/content/360/6393/eaar6343.abstract)

[ZN Kronenberg](https://scholar.google.com/citations?user=HVtr6AIAAAAJ&hl=en&oi=sra), [IT Fiddes](https://scholar.google.com/citations?user=cWfv3CkAAAAJ&hl=en&oi=sra), D Gordon, S Murali… - …, 2018 - science.sciencemag.org

```
SV may effect on expression
* find SV in regulatory region and coding region
* associate SV with expression


Nearly 41% of the genes down-regulated in human radial glia, when compared to chimpanzee radial glial analogs from cerebral organoids, associate with an fhSV and most often as a deletion or a retroposon insertion. 

Long-range sequencing and mapping technologies, such as Strand-seq (49), BAC-based sequencing (63), optical mapping (table S12), and longer-read sequencing (64) will be necessary to sequence resolve such large, more complex SVs.

83% of the ape genomes can be compared in a multiple sequence alignment.
 
36% of human autosomal DNA is subject to incomplete lineage sorting.
 
We detected 614,186 ape deletions, insertions, and inversions, assigning each to specific ape lineages. 
17,789 fixed human-specific insertions and deletions
 
90 that are predicted to disrupt genes and an additional 643 that likely affect regulatory regions, more than doubling the number of human-specific deletions that remove regulatory sequence in the human lineage. 


```





##### [Demography and its effects on genomic variation in crop domestication](https://www.nature.com/articles/s41477-018-0210-1)

[BS Gaut](https://scholar.google.com/citations?user=1nr7TU0AAAAJ&hl=en&oi=sra), [DK Seymour](https://scholar.google.com/citations?user=1nubCLAAAAAJ&hl=en&oi=sra), Q Liu, [Y Zhou](https://scholar.google.com/citations?user=GLvvTpsAAAAJ&hl=en&oi=sra) - Nature Plants, 2018 - nature.com

```shell
# domestication, slow
extended human impacts on domesticated lineages prior to their purposeful cultivation 

# demographiic history, bottleneck, deleterious mutation, genetic load
most crops examined to date have more putatively deleterious alleles than their wild relatives. 

# denographic history ==> SV
* mechanism create SV
* rates of SV origin
* Allele frequency
* phenotype effect
```



most new SV mutations are expected to be either neutral or slightly deleterious in their effects 

During domestication, most SV loci will lose genetic variation and no longer segregate within the crop 

crops will have fewer polymorphic SV loci than their wild relatives 

the SFS of maize CNVs illustrate a slight uptick in high frequency relative to teosintes[78](https://www.nature.com/articles/s41477-018-0210-1#ref-CR78) (consistent with the fixation of CNVs through the domestication bottleneck 。small indels between African rice and its wild progenitor (*Oryza barthii*) exhibit a similar pattern[79](https://www.nature.com/articles/s41477-018-0210-1#ref-CR79) 

Population frequencies of individual SVs between crops and their wild ancestor.  NOT reported yet. 

Some studies have also contrasted population frequencies between crops to identify positively selected SVs (refs 80,81). 

The prevalence of SVs, their evolutionary dynamics and their effects on domestication is in its infancy 



SSR 10E-3

SNP 10E-9

SV ? 



SVs are generated by heterogeneous mechanisms and occur at different rates. Estimates of these rates are important, as rapidly evolving SVs are likely to have weak linkage to nearby SNPs, thus making **traditional SNP-based approaches (such as genome-wide association studies (GWAS)) unlikely to anchor a causative SV.  **



**Indeed, 20% of CNVs in a survey of maize could not be anchored by SNPs82; a slightly higher proportion (~27%) could not be anchored by SNPs in humans83; and two-thirds of transposable elements (TEs) in a population survey of Arabidopsis thaliana were not in linkage disequilibrium with a nearby SNP **



Source of SV:

* TE 10-6 per locus
* gene duplication, 10 Tandem duplication
* To date, SVs have been inferred most often from short-read WGS data based on read coverage 

qxs: what is the length distribution of these SV.



In fruit fly, **identified numerous CNVs that likely contribute to quantitative traits**

To date, the most extensive comparison of plant reference genomes is among Oryza species79,104, but within-species comparisons are still rare within a single crop.  

SV with wild genome. 



 SVs tend to have an outsized effect when they are included in GWAS 

 **We reviewed the primary literature corresponding to 34 of these genes109. In one-third of those genes, the domestication variant was caused by CNVs, large (>1 Kb) indels or similarly large rearrangements (Supplementary Table 2). **

Given these observations, our overriding hypothesis is that most crop adaptation is driven by SVs. If true, most studies of crop adaptation and domestication have ignored the critical genomic variants 







##### [An integrated map of **structural variation** in 2,504 human genomes](https://www.nature.com/articles/nature15394)

PH Sudmant, [T Rausch](https://scholar.google.com/citations?user=fQ1VoZEAAAAJ&hl=en&oi=sra), [EJ Gardner](https://scholar.google.com/citations?user=YHQQCfoAAAAJ&hl=en&oi=sra), RE Handsaker… - Nature, 2015 - nature.com



##### [Structural variations in **plant** genomes](https://academic.oup.com/bfg/article-abstract/13/4/296/2845986)

[RK Saxena](https://scholar.google.com/citations?user=P-ma7nQAAAAJ&hl=en&oi=sra), [D Edwards](https://scholar.google.com/citations?user=AxsOkqYAAAAJ&hl=en&oi=sra)… - Briefings in functional …, 2014 - academic.oup.com

@@ good paper



##### [Structural variation and genome complexity: is dispensable really dispensable?](https://www.sciencedirect.com/science/article/pii/S1369526614000144)

[F Marroni](https://scholar.google.com/citations?user=sIxr9LsAAAAJ&hl=en&oi=sra), [S Pinosio](https://scholar.google.com/citations?user=BtNzIgsAAAAJ&hl=en&oi=sra), [M Morgante](https://scholar.google.com/citations?user=-42meGUAAAAJ&hl=en&oi=sra) - Current opinion in **plant** biology, 2014 - Elsevier



[SVCollector: Optimized sample selection for validating and long-read resequencing of structural variants](https://www.biorxiv.org/content/early/2018/06/08/342386)

Fritz J Sedlazeck, Zachary Lemmon, Sebastian Soyk, William J Salerno, Zachary Lippman, Michael C Schatz

bioRxiv 342386; doi: https://doi.org/10.1101/342386

`一个新方法`



#### SV calling

| BioNano **Access** (B) | <https://bionanogenomics.com/support-page/bionano-access/>   |                                                              |      |
| :--------------------- | :----------------------------------------------------------- | ------------------------------------------------------------ | ---- |
| GROC-SVs (G)           | <https://github.com/grocsvs/grocsvs>                         | [19](https://www.nature.com/articles/s41576-018-0003-4#ref-CR19) |      |
| HiCup (H)              | <https://www.bioinformatics.babraham.ac.uk/projects/hicup/>  | [74](https://www.nature.com/articles/s41576-018-0003-4#ref-CR74) |      |
| LongRanger (G)         | <https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger> |                                                              |      |
| NAIBR (G)              | <https://github.com/raphael-group/NAIBR>                     | [144](https://www.nature.com/articles/s41576-018-0003-4#ref-CR144) |      |
| PBHoney (P)            | <https://sourceforge.net/projects/pb-jelly/>                 | [71](https://www.nature.com/articles/s41576-018-0003-4#ref-CR71) |      |
| SMRT-SV (P)            | <https://github.com/EichlerLab/pacbio_variant_caller>        | [6](https://www.nature.com/articles/s41576-018-0003-4#ref-CR6) |      |
| Sniffles (O and P)     | <https://github.com/fritzsedlazeck/Sniffles>                 | [73](https://www.nature.com/articles/s41576-018-0003-4#ref-CR73) |      |

#### SV consolidation

Combines results from multiple callers to increase sensitivity and reduce false-positive calls 

| MetaSV   | <https://github.com/bioinform/metasv>        | [82](https://www.nature.com/articles/s41576-018-0003-4#ref-CR82) |
| -------- | -------------------------------------------- | ------------------------------------------------------------ |
| SURVIVOR | <https://github.com/fritzsedlazeck/SURVIVOR> | [64](https://www.nature.com/articles/s41576-018-0003-4#ref-CR64) |
| VaPoR    |                                              | [qxs](https://academic.oup.com/gigascience/article/6/8/1/3979348) |



#### Compare SV software

##### [NextSV: a meta-caller for structural variants from low-coverage long-read sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2207-1)

L Fang, J Hu, D Wang, [K Wang](https://scholar.google.com/citations?user=J_veo1sAAAAJ&hl=en&oi=sra) - BMC …, 2018 - bmcbioinformatics.biomedcentral …

```markdown
integrates three aligners and three SV callers

 higher precision and balanced accuracy (F1 score)
 a higher recall
 93.5 to 94.1% for deletions and 87.9 to 93.2% for insertions at 10X coverage
 
 Structural variants (SVs) represent genomic rearrangements (typically defined as longer than 50 bp)
 
 # great example
 Most recently, Merker et al. reported the application of low coverage whole genome PacBio sequencing to identify pathogenic structural variants from a patient with autosomal dominant Carney complex, for whom targeted clinical gene testing and whole genome short-read sequencing were both negative [12]. 
https://www.biorxiv.org/content/biorxiv/early/2016/12/02/090985.full.pdf

 
```









##### [Picky comprehensively detects high-resolution structural variants in nanopore long reads.](https://www.nature.com/articles/s41592-018-0002-6.pdf?origin=ppub)

L Gong, CH Wong, WC Cheng, [H Tjong](https://scholar.google.com/citations?user=EZxs2asAAAAJ&hl=en&oi=sra), F Menghi… - Nature …, 2018 - nature.com

- We validated 100% and 79% of the predicted SVs supported by multiple and single reads, respectively (Supplementary Fig. 4a) 
- Picky recalled 5,246 (66%) of the 7,903 reference DEL calls, comparable  to the 5,251 (66%) and 5,102 (65%) DELs recalled by Sniffles and NanoSV, respectively 
- Fig 2c shows **Sniffles** better than **NanoSV**



##### SV classification:

- inversion (INV), 
- translocation (TLC), 
- tandem duplication (TD), 
- complete tandem duplication within a read segment (TDC), 
- a duplication junction spanning a read segment (TDJ), 
- simple insertion (INS), 
- simple deletion (DEL), and 
- INDEL (both INS and DEL within a single read segment) 



### [A recurrence based approach for validating structural variation using long-read sequencing technology](https://academic.oup.com/gigascience/article-abstract/doi/10.1093/gigascience/gix061/3979348/A-recurrence-based-approach-for-validating)

[X Zhao](https://scholar.google.com/citations?user=IYickbsAAAAJ&hl=en&oi=sra), AM Weber, [RE Mills](https://scholar.google.com/citations?user=BzjA_8kAAAAJ&hl=en&oi=sra) - GigaScience, 2017 - academic.oup.com

```markdown
# VaPoR 

- https://github.com/mills-lab/vapor
- autonomously validates large SV sets using long-read sequencing data 
- VaPoR can interrogate a much larger range of SVs while still matching existing methods in terms of false positive validations 
- providing additional features considering breakpoint precision and predicted genotype 
- VaPoR can run quickly and efficiency without requiring a large processing or assembly pipeline 
- requires relatively low read depth and computing resources
```

